demo/demodb_schema

```diff
@@ -48,9 +48,19 @@ CREATE CLASS history dont_reuse_oid;
 call change_owner('history', 'PUBLIC') on class db_root;
 
 
+CREATE PRIVATE SYNONYM athlete FOR public.athlete;
+CREATE PRIVATE SYNONYM code FOR public.code;
+CREATE PRIVATE SYNONYM [event] FOR public.[event];
+CREATE PRIVATE SYNONYM game FOR public.game;
+CREATE PRIVATE SYNONYM history FOR public.history;
+CREATE PRIVATE SYNONYM nation FOR public.nation;
+CREATE PRIVATE SYNONYM olympic FOR public.olympic;
+CREATE PRIVATE SYNONYM participant FOR public.participant;
+CREATE PRIVATE SYNONYM record FOR public.record;
+CREATE PRIVATE SYNONYM stadium FOR public.stadium;
 
 
-ALTER CLASS stadium ADD ATTRIBUTE
+ALTER CLASS public.stadium ADD ATTRIBUTE
```

Comment:
You have already created a SYNONYM called "stadium" above.
So, can't we just keep this content the same as before without editing it?

Reply:
This statement should work even if there is no synonym.
I think it is clearer to access directly by specifying "PUBLIC" than indirectly through synonym.